lib/util/tXml.js

```diff
@@ -775,3 +803,12 @@ shaka.util.TXml = class {
 };
 
 shaka.util.TXml.knownNameSpaces_ = new Map([]);
+
+
+/**
+ * @typedef {{
+ *   name: string,
+ *   id: ?string
+ * }}
+ */
+shaka.util.TXml.PathNode;
```

Comment:
Shouldn't it be an extern? `shaka.extern.xml.PathNode`?

Reply:
It's not exported nor consumed via public API, but I can move it to externs for consistency with `shaka.extern.xml.Node`. What do you think?

Reply:
For the moment it is fine like this, if we need it in the future, we will change it.